packages/get-platform/src/__tests__/getPlatform.test.ts

```diff
@@ -0,0 +1,140 @@
+      expect(
+        getPlatform({
+          platform,
+          libssl: '1.1.x',
+          arch: 'x64',
+          archFromUname: 'x86_64',
+          familyDistro: 'debian',
+          originalDistro: 'debian',
+          targetDistro: 'debian',
+        }),
+      ).toBe('debian-openssl-1.1.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+
+    it('alpine (alpine), amd64 (x86_64), openssl-3.0.x', () => {
+      expect(
+        getPlatform({
+          platform,
+          libssl: '3.0.x',
+          arch: 'x64',
+          archFromUname: 'x86_64',
+          familyDistro: 'alpine',
+          originalDistro: 'alpine',
+          targetDistro: 'musl',
+        }),
+      ).toBe('linux-musl-openssl-3.0.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+
+    it('alpine (alpine), arm64 (aarch64), openssl-3.0.x', () => {
+      expect(
+        getPlatform({
+          platform,
+          libssl: '3.0.x',
+          arch: 'arm64',
+          archFromUname: 'aarch64',
+          familyDistro: 'alpine',
+          originalDistro: 'alpine',
+          targetDistro: 'musl',
+        }),
+      ).toBe('linux-arm64-openssl-3.0.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+        prisma:warn Prisma only officially supports Linux Alpine on the amd64 (x86_64) system architecture. If you are using your own custom Prisma engines, you can ignore this warning, as long as you've compiled the engines for your system architecture "aarch64".
+        If you are using Prisma on Docker, please refer to https://pris.ly/d/docker-alpine
+      `)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+
+    it('ubuntu (debian), amd64 (x86_64), openssl-undefined', () => {
+      expect(
+        getPlatform({
+          platform,
+          libssl: undefined,
+          arch: 'x64',
+          archFromUname: 'x86_64',
+          familyDistro: 'debian',
+          originalDistro: 'ubuntu',
+          targetDistro: 'debian',
+        }),
+      ).toBe('debian-openssl-1.1.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+        prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
```

Comment:
`Defaulting to "openssl-1.1.x".` should actually be `debian-openssl-1.1.x` here?

Reply:
We're only talking about the default value for `libssl`, so no, `openssl-1.1.x` is the right thing to display in this case

Reply:
This looks confusing as it has the same formatting as our "binaryTarget", not sure if we can do something but also maybe not the most important now.